index.html

```diff
@@ -1167,15 +1167,27 @@ <h3>Malicious Issuers and Verifiers</h3>
 tracking or sharing the presentation of particular credentials.
       </p>
       <p>
-A malicious [=issuer=] might intentionally attack group privacy by creating a
-unique status list per credential issued in order to establish a one-to-one mapping to track
-when a [=verifier=] processes a specific credential. Similarly, they could establish
-another a one-to-one mapping by using a different cryptographic key for every credential
-issued that is tracked in a status list.
+A malicious [=verifier=] might intentionally attack group privacy by sharing
+information from presented credentials with a malicious [=issuer=]. This
+sort of collusion is difficult to detect as it is typically performed via a
+secure communication channel between the [=issuer=] and the [=verifier=].
       </p>
       <p>
-A malicious [=verifier=] might intentionally attack group privacy by sharing
-information from presented credentials with a malicious [=issuer=].
+A malicious [=issuer=] might intentionally attack group privacy by creating a
+unique status list per issued credential, in order to establish a one-to-one
+mapping to track when a [=verifier=] processes each mapped credential. Similarly,
+they could establish a one-to-one mapping by using a different
+cryptographic key for each credential issued that is tracked by a given status list.
+This sort of collusion can be detected by [=holder=] software that serves
+multiple [=holders=] (e.g., a [=holder=] app that runs on a server) if it
+has, for example, an opt-in process that finds that some global identifier(s) 
+used within a [=verifiable credential=] are shared by other credentials.
```

Comment:
I think this needs to be inverted to be correct, responsive to what's stated above, and to flow into the following sentence properly:

```suggestion
This sort of collusion can be detected by [=holder=] software that serves
multiple [=holders=] (e.g., a [=holder=] app that runs on a server) if it
has, for example, an opt-in process that finds that some global identifier(s) 
used within a [=verifiable credential=] are not adequately shared by other
credentials.
```